schema/src/main/jade-fragments/hles_dog_diet.fragment.json

```
@@ -0,0 +1,353 @@
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_primary_diet_component_change_brand_change",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_primary_diet_component_change_new_food_same_brand",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_primary_diet_component_change_other",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_primary_diet_component_change_other_description",
+      "datatype": "string"
+    },
+    {
+      "name": "df_secondary_diet_component_used",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component",
+      "datatype": "integer"
+    },
+    {
+      "name": "df_secondary_diet_component_other_description",
+      "datatype": "string"
+    },
+    {
+      "name": "df_secondary_diet_component_organic",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_grain_free",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_grain_free_past",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_recent",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_months_ago",
+      "datatype": "integer"
+    },
+    {
+      "name": "df_secondary_diet_component_change_allergy_related",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_different_life_stage",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_stop_grain_free",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_health_condition_specific",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_brand_change",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_new_food_same_brand",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_other",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_secondary_diet_component_change_other_description",
+      "datatype": "string"
+    },
+    {
+      "name": "df_treats_frequency",
+      "datatype": "integer"
+    },
+    {
+      "name": "df_treats_commercial_biscuits",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_rawhide",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_bones",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_table_meat",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_table_carbs",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_vegetables",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_homemade_protein",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_homemade_biscuits",
+      "datatype": "boolean"
+    },
+    {
+      "name": "df_treats_pumpkins",
```

```suggestion
      "name": "df_treats_pumpkin",
```
Was this intentional? I'm just comparing things to the mapping sheet and noticed the difference.

---

Good catch